includes/assets/web-stories-block/scss/views/carousel.scss

```diff
@@ -0,0 +1,70 @@
+/* Carousel View Styles */
+.web-stories-list.is-view-type-carousel {
+  position: relative;
+
+  @media all and (min-width: 676px) {
```

Comment:
These media selectors, it would be useful to have these as a variable or something usable, making them easier to change later.

Reply:
Do we have already defined break points or that you prefer, we can use that as well. I'll update this to use custom properties for now.

Reply:
I think you are the first define break points here. Using varibles with a comment should be enough for now.